packages/esm-tb-app/src/api/api.ts

```diff
@@ -0,0 +1,44 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+
+//  Get count of Active DS Cases
+export async function getActiveDsCasesCount() {
+  try {
+    const response = await openmrsFetch('ws/rest/v1/mamba/report?report_id=total_active_ds_cases');
```

Comment:
Assuming this is the standard way to get mamba reports, this can be changed to a reusable hook with SWR and added to the commons lib as part of the refactor. You add some parameters like report_id and filters (if supported) 

For the start, something like this can work. We can then reuse it in other clinical dashboards to replace the static data in the tiles

```
export function useMambaReport(reportId: string)
```